--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -55,7 +55,10 @@ set_target_properties(madx PROPERTIES LI
 
 # set library version:
 set_target_properties(madx     PROPERTIES VERSION ${MADX_VERSION})
-set_target_properties(madx     PROPERTIES CXX_STANDARD 11)
+
+# Can't use CXX_STANDARD prior to cmake 3.1:
+# set_target_properties(madx     PROPERTIES CXX_STANDARD 11)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
 
 # adding an executable:
